marketplace-and-integrations/packages/authorized-services.md

```diff
@@ -0,0 +1,776 @@
+---
+
+# Authorized Services
+
+## Introduction
+
+**Umbraco Authorized Services** is an Umbraco package designed to reduce the effort needed to integrate third party solutions into Umbraco solutions.  Many such services require an OAuth flow fir authentication and authorization.  Working with these services requires a fair bit of plumbing code to handle creating an authorized connection.  This is necessary before the developer working with the service can get to using the provided API to implement the business requirements.
+
+There are similarities to the flow that needs to be implemented for different services.  Steps include:
+
+- Redirecting to an authentication endpoint.
+- Handling the response including an authentication code and exchanging it for an access token.
+- Securely storing the token.
+- Including the token in API requests.
+- Serializing requests and deserializing the API responses.
+- Handling cases where the token has expired and obtaining a new one via a refresh token.
+
+There are though also differences, across request and response structures and variations in the details of the flow itself.
+
+The package tries to implement a single, best practice implementation of working with OAuth. For particular providers the specific flow required can be customized via configuration or code.
+
+## Features
+
+For the solution developer, the Umbraco Authorized Services offers two primary features.
+
+Firstly there's a tree available in the _Settings_ section of the backoffice, called _Authorized Services_. The tree shows the list of services based on the details provided in configuration.
+
+![authorized-services-tree](images/authorized-services-tree.png)
+
+Each tree entry has a management screen where an administrator can authenticate with an app that has been setup with the service. The status of each service is shown on this screen. When authorized, the authentication and authorization flow has been completed and an access token stored.
+
+![not-authorized-screen](images/not-authorized-screen.png)
+
+![authorized-screen](images/authorized-screen.png)
+
+Secondly, the developer has access to an interface - `IAuthorizedServiceCaller` - that they can inject instances of and use to make authorized requests to the service's API.
+
+Using a settings screen the administrator can review the service configuration.
+
+![settings-screen](images/settings-screen.png)
+
+## Usage
+
+### App Creation
+
+Services that this package are intended to support will offer an OAuth authentication and authorization flow against an "app". The developer will need to create this "app" with the service.  By doing this information such as the "client ID" and "client secret" can be applied in configuration.
+
+### Installation
+
+The package can be installed into your Umbraco solution from [NuGet](https://www.nuget.org/packages/Umbraco.AuthorizedServices).
```

Comment:
"The package _can be_ installed ... "

Are there other methods for installing the package?
And if so, should we mention them? 🤔

Reply:
No, not really.  I guess you could compile the source code yourself, but it's not really sensible.  I've updated "can" to "should".